receiver/receivertest/contract_checker.go

```diff
@@ -437,3 +506,42 @@ func CreateOneLogWithID(id UniqueIDAttrVal) plog.Logs {
 	)
 	return data
 }
+
+func CreateEveryMetricTypeWithID(id UniqueIDAttrVal) pmetric.Metrics {
```

Comment:
I don't think we should use the same id across several metrics. The ID is supposed to be unique per datapoint

Reply:
alright, changed

Reply:
This isn’t addressed. Why do we need every metric type with the same ID? Can this method take metric type instead?

Reply:
Sorry, bad push. Should be good now.